mkdocs/docs/HPC/getting_started.md

```diff
@@ -0,0 +1,209 @@
+# Getting Started
+
+Welcome to the "Getting Started" guide. This chapter will lead you through the initial steps of logging into the HPC-system and submitting your very first job. We'll also walk you through the process step by step using a practical example.
+
+In addition to this chapter, you might find these  [slides](https://www.ugent.be/hpc/en/training/2023/introhpc202306) to be useful resources.
```

Comment:
avoid pointing to training session of a specific date, better point to https://www.ugent.be/hpc/en/training/introhpcugent-recording instead (which tells you where to find the slides)